e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -72,6 +74,38 @@ describe('Messaging', () => {
         cy.get('#post_textbox').should('be.focused');
     });
 
+    it('M17449 - Focus will move to main input box after a new channel has been opened', () => {
+        //# Click on Town-Square channel
+        cy.get('#sidebarItem_town-square').click({force: true});
+
+        //# Open the reply thread on the most recent post on Town-Square channel
+        cy.clickPostCommentIcon();
+
+        //# Place the focus inside the RHS input box
+        cy.get('#reply_textbox').focus().should('be.focused');
+
+        //# Use CTRL+K or CMD+K to open the channel switcher depending on OS
+        cy.typeCmdOrCtrl().type('K', { release: true });
+
+        //* Verify channel switcher is visible
+        cy.get('#quickSwitchHint').should('be.visible');
+
+        //# Type channel name 'sint'
+        cy.get('#quickSwitchInput').type('sint');
+
+        //# Wait 500 ms
+        cy.wait(TIMEOUTS.TINY);
+
+        //# Press Enter to select the channel
+        cy.get('#quickSwitchInput').type('{enter}');
```

Comment:
All the above lines can be combined in a single step. Something like this:
cy.get('#quickSwitchInput').type('sint').wait(TIMEOUTS.TINY).type('{enter}');